Modules/Core/ImageFunction/include/itkInterpolateImageFunction.h

```diff
@@ -124,6 +127,13 @@ class ITK_TEMPLATE_EXPORT InterpolateImageFunction:
     return ( static_cast< RealType >( this->GetInputImage()->GetPixel(index) ) );
   }
 
+  /** Get the radius required for interpolation.
+   *
+   * This defines the number of surrounding pixels required to interpolate at
+   * a given point.
+   */
+  virtual SizeType GetRadius() const = 0;
```

Comment:
This requires all derived interpolators to implement this method ( not too bad of a thing ). An alternative is to return the size of the image, so that the default usage would indicate the whole image is needed.

Reply:
Implementing this is not hard. Let's require all interpolators to do it. But we should add a note to migration guide.